.github/workflows/release.yaml

```diff
@@ -134,7 +134,12 @@ jobs:
         uses: crazy-max/ghaction-import-gpg@v1
         env:
           GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
-          PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }} 
+          PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
+          
+      - name: Fetch Latest Commit SHA For Apicurio Registry
+        run: |
+          cd studio
+          echo "latest_commit_sha=$(git log -n 1 --pretty=format:"%H")" >> $GITHUB_ENV
```

Comment:
Copy/paste error here I think - there's no `studio` here. :)

Reply:
@EricWittmann fixed the typo.